Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -239,6 +239,39 @@ class AppStoreConnectService {
         _ = try expireBuildOperation.execute(with: requestor).await()
     }
 
+    func listBuilds(
+        filterBundleIds: [String],
+        filterExpired: [String],
+        filterPreReleaseVersions: [String],
+        filterBuildNumbers: [String],
+        filterProcessingStates:[ListBuilds.Filter.ProcessingState],
+        filterBetaReviewStates: [String], limit: Int?
+    ) throws -> [Build] {
+
+        var filterAppIds: [String] = []
+
+        if !filterBundleIds.isEmpty {
+            let appsOperation = GetAppsOperation(options: .init(bundleIds: filterBundleIds))
+            filterAppIds = try appsOperation.execute(with: requestor).await().map(\.id)
+        }
+
+        let listBuildsOperation = ListBuildsOperation(
+            options: .init(
+                filterAppIds: filterAppIds,
+                filterExpired: filterExpired,
+                filterPreReleaseVersions: filterPreReleaseVersions,
+                filterBuildNumbers: filterBuildNumbers,
+                filterProcessingStates: filterProcessingStates,
+                filterBetaReviewStates: filterBetaReviewStates,
+                limit: limit
+            )
+        )
+
+        let output = try listBuildsOperation.execute(with: requestor).await()
+        let builds = output.map { Build($0.build, $0.relationships) }
+        return builds
```

Comment:
You can probably get away with a function reference here:
```suggestion
        return output.map(Build.init)
```